### PR TITLE
Handle dot index

### PR DESCRIPTION
**Pull request by arielkru**
Handle index pattern `.int` in addition to `[int]`

Solves https://github.com/bridgecrewio/checkov/issues/355

@aoskotsky-amplify

### CODE REVIEW

**Comment by schosterbarak** [comment]
Hi @jconstance-amplify , @aoskotsky-amplify
any chance you can take a look at this one?

File: hcl2/hcl2.lark [hunk 1]
```diff
@@ -66,7 +66,7 @@ index_expr_term : expr_term index
 get_attr_expr_term : expr_term get_attr
 attr_splat_expr_term : expr_term attr_splat
 full_splat_expr_term : expr_term full_splat
-index : "[" new_line_or_comment? expression new_line_or_comment? "]"
+index : "[" new_line_or_comment? expression new_line_or_comment? "]" | "." DECIMAL
```
Comment (hi117) [inline hcl2/hcl2.lark:69]: This should be DECIMAL+ since there might be lists of length greater than 10

**Comment by rarruda** [comment]
It would be awesome if it was possible to get a release with this fix... Would solve some issues I am currently facing when using this library.

**Comment by aoskotsky-amplify** [comment]
Hey, thanks for the PR. I pulled it and cleaned a few things up in a new one https://github.com/amplify-education/python-hcl2/pull/44. I'll have it merged soon

**Comment by aoskotsky-amplify** [comment]
Should be fixed now with #44. Let me know if there are any issues

**Pull request closed without merging**